README_hu.md

```diff
@@ -9,6 +9,22 @@ A sablon magyar és angol nyelvű dokumentumok elkészítését is támogatja (l
 
 ## Fordítás
 
+A fordításhoz tetszőleges fejlesztő környezet is használható (pl. [TexStudio](https://www.texstudio.org/)), ugyanezen utasítások kiadásával.
+
+### Build rendszer használatával
+
+```bash
+# Mindent lefordít automatikusan, a megfelelő sorrendben.
+# (GNU make kell hozzá!)
+make
```

Comment:
While the default target also leads to `thesis`, the English and Hungarian readme should be consistent, so use `make thesis` here as well.